.gitignore

```diff
@@ -22,3 +22,4 @@ index.html
 .cache
 .pytest_cache
 pyvenv.cfg
+.idea
```

Comment:
Added this because I use PyCharm, making sure no IDE settings are being committed